### PR TITLE
backport: f3: Upgrade to go-f3 `v0.0.5` (#12279)

### PR DESCRIPTION
**Pull request by jennijuju**
backported  12279 and updated changelog for f3

### CODE REVIEW

**Comment by rjan90** [comment]
Rebased the PR onto the latest `phi/v1280-stable`.

File: CHANGELOG.md [hunk 1]
```diff
@@ -65,6 +65,14 @@ We recommend node operators (who haven't enabled splitstore discard mode) that d
...
 
+## Fast Finality for Filecoin (f3) soft launch
+
+We are one step closer to reduce Filecoin's finality from 7.5 hours to a couple of minutes, you can checkout the [FIP](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0086.md) for more details.  Changing the consensus protocol is not trivial, and the f3 implementation team has designed a [passive testing plan to verify the protocol](https://github.com/filecoin-project/go-f3/issues/213) and give time for client implementation teams to integrate and test F3 before it is fully activated in the network consensus. That said, lotus team has implemented f3 & the manifest for passive testing in this release, and we would like to ask node operators especially storage providers to participate in the testing by participate in F3 on mainnet (which is enabled by default tin this release)! We will keep updating [this discussion](https://github.com/filecoin-project/lotus/discussions/12287) to capture "what can you expect" & testing status. If you notice any unexpected behaviour caused by f3, please do not feel hesitant to reach out to us in #fil-fast-finality.
```
Comment (Kubuxu) [inline CHANGELOG.md:70]:
```suggestion
We are one step closer to reduce Filecoin's finality from 7.5 hours to a couple of minutes, you can checkout the [FIP](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0086.md) for more details.  Changing the consensus protocol is not trivial, and the f3 implementation team has designed a [passive testing plan to verify the protocol](https://github.com/filecoin-project/go-f3/issues/213) and give time for client implementation teams to integrate and test F3 before it is fully activated in the network consensus. That said, the lotus team has implemented f3 & the manifest for passive testing in this release, and we would like to ask node operators, especially storage providers, to participate in the testing by participating in F3 on the mainnet (which is enabled by default in this release)! We will keep updating [this discussion](https://github.com/filecoin-project/lotus/discussions/12287) to capture "what can you expect" & testing status. If you notice any unexpected behaviour caused by f3, please do not hesitate to reach out to us in #fil-fast-finality. 
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -65,6 +65,14 @@ We recommend node operators (who haven't enabled splitstore discard mode) that d
...
 
+## Fast Finality for Filecoin (f3) soft launch
+
+We are one step closer to reduce Filecoin's finality from 7.5 hours to a minute, you can checkout the [FIP](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0086.md) for more details.  Changing the consensus protocol is not trivial, and the f3 implementation team has designed a [passive testing plan to verify the protocol](https://github.com/filecoin-project/go-f3/issues/213) and give time for client implementation teams to integrate and test F3 before it is fully activated in the network consensus. That said, the lotus team has implemented f3 & the manifest for passive testing in this release, and we would like to ask node operators, especially storage providers, to participate in the testing by participating in F3 on the mainnet (which is enabled by default in this release)! We will keep updating [this discussion](https://github.com/filecoin-project/lotus/discussions/12287) to capture "what can you expect" & testing status. If you notice any unexpected behaviour caused by f3, please do not hesitate to reach out to us in #fil-fast-finality.
```
Comment (rjan90) [inline CHANGELOG.md:70]:
```suggestion
We are one step closer to reduce Filecoin's finality from 7.5 hours to a minute or so, you can checkout the [FIP](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0086.md) for more details.  Changing the consensus protocol is not trivial, and the f3 implementation team has designed a [passive testing plan to verify the protocol](https://github.com/filecoin-project/go-f3/issues/213) and give time for client implementation teams to integrate and test F3 before it is fully activated in the network consensus. That said, the lotus team has implemented f3 & the manifest for passive testing in this release, and we would like to ask node operators, especially storage providers, to participate in the testing by participating in F3 on the mainnet (which is enabled by default in this release)! We will keep updating [this discussion](https://github.com/filecoin-project/lotus/discussions/12287) to capture "what can you expect" & testing status. If you notice any unexpected behaviour caused by f3, please do not hesitate to reach out to us in #fil-fast-finality. 
```

File: CHANGELOG.md (hunk 1 above)
Comment (rjan90) [reply CHANGELOG.md]: Ack! Added in the "or so" in: https://github.com/filecoin-project/lotus/pull/12288/commits/1bde5f6c2d3c2973478d181a3b411dbe36a0c885

**Pull request merged**